### ISSUE TITLE
How to make yourself invisible, why does the server accept SUS colors of skin??? a.k.a 1upping issue #6478

### ISSUE THREAD

**Issue by Kehren** [unitystation/unitystation#6479]
As predicted the server accepts whatever character settings i throw at it (quote issue #6478), even ALPHA color values lmao

![image](https://user-images.githubusercontent.com/79835205/117656130-a0777880-b1a0-11eb-97fa-02213a8bb546.png)
![image](https://user-images.githubusercontent.com/79835205/117657958-d7e72480-b1a2-11eb-93af-ef73ab2412b8.png)
![image](https://user-images.githubusercontent.com/79835205/117657981-dd446f00-b1a2-11eb-83da-9a0a3eebf897.png)
![image](https://user-images.githubusercontent.com/79835205/117658130-0c5ae080-b1a3-11eb-9f76-61eebba54f0b.png)
![image](https://user-images.githubusercontent.com/79835205/117658499-76738580-b1a3-11eb-8b18-cea2f60dfb2d.png)

One can make themselves fully invisible, just for the sake of demonstration i didnt make myself fully invisible

HOW TO REPLICATE
haha alpha go brr
In short i just sent the server color values with Alpha channels
![image](https://user-images.githubusercontent.com/79835205/117659104-33fe7880-b1a4-11eb-9f74-0db2e11e44ab.png)
pretty sure this works because 
![image](https://user-images.githubusercontent.com/79835205/117659516-b5eea180-b1a4-11eb-8e12-f4277deb25d7.png)
tryparsehtmlstring accepts alpha values
and its the only thing checking color on the servers side

discord: Jakesolly#0966

**Comment by Kehren** [comment]
Apparently this also has the side effect of making you invincible to interaction, as in, noone can disarm
also noone can inspect, so you are litterary invisible to the world

**Comment by jamessimo** [comment]
> Apparently this also has the side effect of making you invincible to interaction, as in, noone can disarm
> also noone can inspect, so you are litterary invisible to the world

Pretty sure that’s because unity raycasts visible pixels.

**Comment by NoooneyDude** [comment]
Fixed by #6511,
